pkg/controller/utils/machine_helper.go

```diff
@@ -0,0 +1,109 @@
+package utils
+
+import (
+	"context"
+	"reflect"
+	"strings"
+
+	machineapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	"github.com/prometheus/common/log"
```

Comment:
How come we're using this for logging and not `sigs.k8s.io/controller-runtime/pkg/log`?

Reply:
This switched on me when I saved the file, and I'm not sure why

Reply:
Spoooooooky!